ProcessList.c

```diff
@@ -25,6 +29,9 @@ ProcessList* ProcessList_init(ProcessList* this, const ObjectClass* klass, Users
    // tree-view auxiliary buffer
    this->processes2 = Vector_new(klass, true, DEFAULT_SIZE);
 
+   this->displayTreeSet = Hashtable_new(4096, false);
```

Comment:
Probably use better initial size than 4096 (`Vector_size(this->processes)  0.7 /* load factor */` ?)

Reply:
Probably `Vector_size(this->processes) * 1.5` (`size = Vector_size(this->processes; size += size >> 1;`)